module-gui/gui/widgets/text/Text.cpp

```diff
@@ -360,6 +360,19 @@ namespace gui
         this->inputMode = mode;
     }
 
+    bool Text::isInputMode(InputMode::Mode mode)
+    {
+        if (inputMode == nullptr) {
```

Comment:
I believe you can simplify this function eg.
`        if (inputMode != nullptr && inputMode->is(mode)) {
            return true;
        }
        return false;`

Reply:
Yes, but now the structure of this method looks similar to the methods around this one
(and we can log situation when `inputMode` is nullptr)

Reply:
If it is important to catch situations when the `inputMode` is `nullptr` that's ok. You can leave it as it is. ;)